libindy/src/services/wallet/default.rs

```diff
@@ -403,4 +430,30 @@ mod tests {
 
         TestUtils::cleanup_indy_home();
     }
-}
\ No newline at end of file
+
+    #[test]
+    fn default_wallet_create_encrypted() {
```

Comment:
It would be nice to provide some additional test cases:

1. Trying to open unencrypted database with some key
2. Trying to open encrypted database with invalid key
3. Trying to use key without Key Derivation (in format "x'2DD29CA851E7B56E4697B0E1F08507293D761A05CE4D1B628663F411A8086D99'")

Reply:
Ideally, we need to add these tests to wrappers too, but as it backward compatible change we can merge it without wrappers update, but for Rust code, we need these tests.